packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -64,7 +65,7 @@ export class GutenbergEditorPage {
 	 *
 	 * @returns {Promise<Frame>} iframe holding the editor.
 	 */
-	private async getEditorFrame(): Promise< Frame > {
+	async getEditorFrame(): Promise< Frame > {
```

Comment:
I'd prefer that this method remain `private`. Methods needing to obtain the frame should use `waitUntilLoaded` which is accessible publicly.